components/stratum/stratum_api.c

```diff
@@ -275,29 +277,29 @@ void STRATUM_V1_free_mining_notify(mining_notify * params)
     free(params);
 }
 
-int _parse_stratum_subscribe_result_message(const char * result_json_str, char ** extranonce, int * extranonce2_len)
+int _parse_stratum_subscribe_result_message(const char * POOL_TAG, const char * result_json_str, char ** extranonce, int * extranonce2_len)
```

Comment:
Is this function unused, except in a commented-out piece of test code?

Reply:
Yep looks unused. Removed it.